youtube_dl/extractor/hrti.py

```diff
@@ -104,7 +104,7 @@ class HRTiIE(HRTiBaseIE):
                         (?:
                             hrti:(?P<short_id>[0-9]+)|
                             https?://
-                                hrti\.hrt\.hr/\#/video/show/(?P<id>[0-9]+)/(?P<display_id>[^/]+)?
+                                hrti\.hrt\.hr/(\#/)?video/show/(?P<id>[0-9]+)/(?P<display_id>[^/]+)?
```

Comment:
Don't capture groups you don't use.

Reply:
The group is there for optionality. I wanted to match only `...hrti.hrt.hr/#/video...` and `...hrti.hrt.hr/video...`
Then `\#?/?` (this catches `...hrti.hrt.hr//video...` too) or don't capture with `/#/` (but I'm not sure if there are still links with it). Suggestions?

Reply:
You don't get it. Group is fine, but it **should not** capture.